src/Tribe/Assets.php

```diff
@@ -77,6 +77,22 @@ public function enqueue_scripts() {
 					],
 				]
 			);
+
+			// @todo: Remove this once we solve the common breakpoints vs container based.
+			tribe_asset(
+				$tickets_main,
+				'tribe-common-responsive',
+				'common-responsive.css',
+				[ 'tribe-common-skeleton-style' ],
+				null,
+				[
+					'groups' => [
```

Comment:
:no_entry_sign: Each line in an array declaration must end in a comma

<sub>hash: 6545b98b657a76bf77e1dce02516a93e</sub>